### ISSUE TITLE
Diablo 2 Resurrected cursor has a squiggle or line artifact underneath it

### ISSUE THREAD

**Issue by mwpow3ll** [HansKristian-Work/vkd3d-proton#822]
It's mostly visible at the Main / Character selection screen

Installed D2 via the following method:

- Download battle.net installer from Blizzard's website
- Add battle.net installer executable to Steam as a Non-Steam game
    - Set Steam to use Proton experimental when launching
- Click Play
- Step through the Battle.net installer process
    - Download D2 Resurrected from within Battle.net launcher
- Modify the Non-Steam game link to point to the newly installed Diablo 2 Resurrected Battle.net launcher
- Click Play

System info:
- CPU: Intel i7-8700k
- RAM: 64GB DDR4
- GPU: Nvidia 1080 Ti (running driver version 470.63)
- Fedora 34
- Xorg: 1.20.11
- Steam: Sep 17 2021, at 4:30:38
- Proton: experimental-6.3-20210924

**Comment by Mershl** [comment]
Do you also see line artifacts near fires/lamps/... like in this screencast: https://streamable.com/qujizr ?

**Comment by mwpow3ll** [comment]
> Do you also see line artifacts near fires/lamps/... like in this screencast: https://streamable.com/qujizr ?

No, I do not see that issue. I suspect your artifact issue a compatibility issue with vkd3d, mesa, and AMD.

My understanding is that Nvidia rolls their own openGL / VK driver instead of relying on Mesa which is probably why I don't see it.

**Comment by evilch3n** [comment]
> Do you also see line artifacts near fires/lamps/... like in this screencast: https://streamable.com/qujizr ?

i have that excat same effect too, i thought my GPU-Unit bricked or something haha

**Comment by Mershl** [comment]
> > Do you also see line artifacts near fires/lamps/... like in this screencast: https://streamable.com/qujizr ?
> 
> i have that excat same effect too, i thought my GPU-Unit bricked or something haha

If you are on AMDGPU + mesa driver can you please add a comment with your hardware spec + used mesa version to this issue report: https://gitlab.freedesktop.org/mesa/mesa/-/issues/5402? Thank you very much. This should help narrow down the issue.

Sorry for hijacking this thread a bit - Github does not allow send private messages...

**Comment by mwpow3ll** [comment]
> Sorry for hijacking this thread a bit - Github does not allow send private messages...

I'm cool with it ;-)

I'm glad to see your issue gaining traction.
